features/zoeken/met-geslachtsnaam-voornamen-en-gemeente-van-inschrijving/overzicht.feature

```diff
@@ -0,0 +1,73 @@
+      Gegeven de persoon 'Pieter' met burgerservicenummer '000000024'
+      * heeft de volgende gegevens
+        | geslachtsnaam (02.40) | voornamen (02.10) |
+        | Maassen               | Pieter            |
+      En 'Pieter' is 5 jaar geleden ingeschreven op adres 'A1'
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met geslachtsnaam '<geslachtsnaam>', voornamen '<voornamen>' en gemeente van inschrijving '0599'
+      Dan wordt 'Pieter' gevonden
+
+      Voorbeelden:
+        | geslachtsnaam | voornamen |
+        | Maassen       | Pieter    |
+        | MAASSEN       | PIETER    |
+        | MAASSEN       | pieter    |
+
+    Scenario: de te vinden persoon heeft een geslachtsnaam van twee karakters lang
+      Gegeven de persoon 'Miriam' met burgerservicenummer '000000025'
+      * heeft de volgende gegevens
+        | geslachtsnaam (02.40) | voornamen (02.10) |
+        | Os                    | Miriam            |
+      En 'Miriam' is 5 jaar geleden ingeschreven op adres 'A1'
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met geslachtsnaam 'Os', voornamen 'Miriam' en gemeente van inschrijving '0599'
+      Dan wordt 'Miriam' gevonden
+
+  Regel: de optionele 'voorvoegsel' parameter (niet hoofdlettergevoelig) kan worden gebruikt om de zoek criteria aan te scherpen
+
+    Scenario: geslachtsnaam, voorvoegsel, voornamen en gemeente waar de personen zijn ingeschreven komen overeen met de opgegeven geslachtsnaam, voorvoegsel, voornamen en gemeente van inschrijving parameters
+      Gegeven de persoon 'Pieter' met burgerservicenummer '000000024'
+      * heeft de volgende gegevens
+        | geslachtsnaam (02.40) | voorvoegsel (02.30) | voornamen (02.10) |
+        | Maassen               | van                 | Pieter            |
+      En 'Pieter' is 5 jaar geleden ingeschreven op adres 'A1'
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met geslachtsnaam 'Maassen', voornamen 'Pieter' en gemeente van inschrijving '0599' en parameters
+        | voorvoegsel    |
+        | <voorvoegsels> |
+      Dan wordt 'Pieter' gevonden
+
+      Voorbeelden:
+        | voorvoegsels |
+        | van          |
+        | Van          |
+        | VAN          |
+
+  Regel: de optionele 'inclusiefOverledenPersonen' parameter moet worden opgegeven om overleden personen te kunnen vinden
+
+    @info-api
+    Scenario: geslachtsnaam, voornamen en de gemeente waar de overleden persoon staat ingeschreven komen overeen met de opgegeven geslachtsnaam, voornamen en gemeente van inschrijving parameters
+      Gegeven de persoon 'Jan' met burgerservicenummer '000000026'
+      * heeft de volgende gegevens
+        | geslachtsnaam (02.40) | voornamen (02.10) |
+        | Jansen                | Jan               |
+      En 'Jan' is 5 jaar geleden ingeschreven op adres 'A1'
+      En 'Jan' is op 23-02-2022 overleden
```

Comment:
over twee jaar wordt dit een raar scenario, omdat de persoon dan wordt ingeschreven op het adres nadat deze is overleden

Reply:
ook de datum inschrijving op adres een exacte datum maken (die voor 23-02-2022 ligt)

Reply:
aangepast